awl/utils.cpp

```diff
@@ -59,7 +59,7 @@ QString pitch2string(int v)
       {
       if (v < 0 || v > 127)
             return QString("----");
-      int octave = (v / 12) - 2;
```

Comment:
1

Reply:
I'm a bit confused about your comment - I changed the int octave to subtract 1 instead of 2, so int octave = (v / 12) - 1 so then middle C with a value of 60 would get the correct octave number C4